### ISSUE TITLE
Update dataQuality report evaluationMethod element

### ISSUE THREAD

**Issue by hmaier-fws** [adiwg/mdJson-schemas#274]
Update `metadata > dataQuality > report > evaluationMethod` as follows:
  
  - Change description to: "Description of the evaluation method and procedure applied"
  - `evaluationMethod` translations:
    - ISO 19115-2: MD_Metadata > DQ_DataQuality > report > DQ_EvaluationMethod
    - FGDC CSDGM:
    - DCAT: no translation

  - Add translations for `evaluationMethod` elements:
    - [ ] type
    - [ ] dateTime
    - [ ] methodDescription
    - [x] evaluationProcedure
    - [ ] referenceDocument
    - [ ] evaluationMethodType
    - [ ] deductiveSource
    - [ ] samplingScheme
    - [ ] lotDescription
    - [ ] samplingRatio
 
# Translations for `evaluationMethod` elements

**type**:
  - ISO 19115-2: MD_Metadata > DQ_DataQuality > report > evaluationMethod > type
  - FGDC CSDGM:
  - DCAT: no translation

**evaluationMethodType**:
  - ISO 19115-2: MD_Metadata > DQ_DataQuality > report > evaluationMethod > evaluationMethodType [DQ_EvaluationMethodTypeCode]
  - FGDC CSDGM:
  - DCAT: no translation
  
# Add rules for evaluationMethod 
  - deductiveSource: required if type [ DQ_IndirectEvaluation ]
  - samplingScheme: required if type [ DQ_SampleBasedInspection ]
  - lotDescription: required if type [ DQ_SampleBasedInspection ]
  - samplingRatio: required if type [ DQ_SampleBasedInspection ]

DQ_EvaluationMethodTypeCode (see: https://standards.iso.org/iso/19157/resources/Codelists/cat/codelists.html)

For details see original issue: #226

**Comment by hmaier-fws** [comment]
@dwalt should we add translations for all of the above elements? I assume most are not supported by CSDGM.
  - Should we just add a translation for a parent element?
  - Would it be useful to inform users that there is no translation?

**Comment by dwalt** [comment]
@hmaier-fws Yes, this is one of the criteria to evaluate DQ, identifying methods used to determine quality and should translate to ISO. Methods will not translate to CSDGM. We have many elements that don't translate to CSDGM. Documenting the translations support in the schema description will provide that documentation. I don't understand the idea of a parent element. We did draft a mapping of ISO DQ to CSDGM DQ but given the complexity and no clear use case it was abandoned.
